content-blocker-lib-ios/js/TrackingProtectionStats.js

```diff
@@ -150,11 +150,24 @@ function install() {
         if (!_tpErrorHandler.get(this)) {
           // If this `Image` instance fails to load, we can assume
           // it has been blocked.
-          var tpErrorHandler = function() {
+          let tpErrorHandler = () => {
             sendMessage(this.src);
           };
+
           _tpErrorHandler.set(this, tpErrorHandler);
-          this.addEventListener("error", tpErrorHandler);
+
+          // Unfortunately, we need to wait a tick before attaching
+          // our event listener otherwise we risk crashing the
+          // WKWebView content process (Bug 1489543).
+          requestAnimationFrame(() => {
+            // Check if the error has already occurred before
+            // we had a chance to attach our event listener.
+            if (this.src && this.complete && this.naturalWidth === 0) {
+              tpErrorHandler();
+            }
```

Comment:
return needed for this block?

Reply:
I don't think because we still want to attach our event listener in case the same `Image` instance gets reused again to try and load another image (by re-setting the `src` property).